### PR TITLE
Core 404 bug notary verification

### PR DESCRIPTION
**Pull request by karladler**

### CODE REVIEW

File: dapps/organizations/src/components/detail/info/info-content.vue [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  of it on other blockchains than evan.network.
+
+  For more information, please contact evan GmbH at this address:
+  https://evan.network/license/
+*/
+
+<template>
+  <div>
+    <h4>{{'_org.ident.notary.request.why.title' | translate}}</h4>
+    <p class="mb-4">{{ $t('_org.ident.notary.request.why.description', { accountId: $route.params.address }) }}
```
Comment (Tschuck) [inline dapps/organizations/src/components/detail/info/info-content.vue]: the p element includes multiple elements, missing enter before `<p class="mb-4">`

File: dapps/organizations/src/components/detail/info/info-content.vue [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      <a href="https://evannetwork.github.io/docs/first_steps/power_apps/notary-verification.html"
+        class="text-link" target="_blank" rel="noreferer noopener">
+        {{'_org.ident.notary.request.why.link' | translate}}
+      </a>
+    </p>
+
+    <h4>{{'_org.ident.notary.request.who.title' | translate}}</h4>
+      <p class="mb-4">{{'_org.ident.notary.request.who.description' | translate}}
```
Comment (Tschuck) [inline dapps/organizations/src/components/detail/info/info-content.vue]: the p element includes multiple elements, missing enter before `<p class="mb-4">`

File: dapps/organizations/src/i18n/en.ts [hunk 3]
```diff
@@ -222,11 +222,11 @@ const i18n: any = {
         },
         "request-error": "Request could not be loaded!",
         "status": {
-          "accepted": "notarially verified",
```
Comment (Tschuck) [inline dapps/organizations/src/i18n/en.ts:225]: Some more translation keys in this file includes `notarial verification`, should be `notary verification`

File: dapps/organizations/src/components/detail/info/info.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  max-height: calc(90vh - 140px - 32px);
+  padding: 0 3rem;
+
+  .modal-header h5 {
+    margin-left: 3rem;
+  }
+}
+
+/* Theme overwrite */
+.evan.theme-evan {
```
Comment (Tschuck) [inline dapps/organizations/src/components/detail/info/info.scss]: The new seperated components like `info-content` includes a evan-theme overwrite. Should we move this to one `global` style for the organization application? Would be possible by creating a `index.scss` in the root folder, that is required by the `index.ts` file and adding a `organization` class to the root component.

    ```css
    .evan.theme-evan {
      h4, h5, .bold {
        font-weight: bold;
      }
    }
    ```

File: dapps/organizations/src/components/detail/info/info-content.scss [hunk 5]
```diff
@@ -0,0 +1,16 @@
+@function cssVar($name) {
```
Comment (Tschuck) [inline dapps/organizations/src/components/detail/info/info-content.scss]: `cssVar` can be imported via `@import '~@evan.network/ui/src/style/utils';`

File: dapps/organizations/src/components/detail/info/info-content.scss [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+.step > div {
+  height: 100%;
+  padding: 12px;
+  background: cssVar('bg-level-3');
+}
+
+/* Theme overwrite */
+.evan.theme-evan {
```
Comment (Tschuck) [inline dapps/organizations/src/components/detail/info/info-content.scss]: The new seperated components like `info-content` includes a evan-theme overwrite. Should we move this to one `global` style for the organization application? Would be possible by creating a `index.scss` in the root folder, that is required by the `index.ts` file and adding a `organization` class to the root component.

    ```css
    .evan.theme-evan {
      h4, h5, .bold {
        font-weight: bold;
      }
    }
    ```

File: dapps/organizations/src/components/detail/info/info.scss [hunk 7]
```diff
@@ -0,0 +1,18 @@
+.modal .modal-dialog .modal-content {
```
Comment (Tschuck) [inline dapps/organizations/src/components/detail/info/info.scss:28]: missing the copyright header

File: dapps/organizations/src/components/detail/info/info-content.scss (hunk 5 above)
Comment (Tschuck) [inline dapps/organizations/src/components/detail/info/info-content.scss]: missing the copyright header

File: dapps/organizations/src/components/detail/identifications/notary/request/request.scss [hunk 8]
```diff
@@ -1,18 +1,19 @@
...
-    height: 100%;
-    padding: 12px;
+  .modal-header h5 {
+    margin-left: 3rem;
+  }
+}
 
-    background: #f5f6fa;
+/* Theme overwrite */
+.evan.theme-evan {
```
Comment (Tschuck) [inline dapps/organizations/src/components/detail/identifications/notary/request/request.scss]: The new seperated components like `info-content` includes a evan-theme overwrite. Should we move this to one `global` style for the organization application? Would be possible by creating a `index.scss` in the root folder, that is required by the `index.ts` file and adding a `organization` class to the root component.

    ```css
    .evan.theme-evan {
      h4, h5, .bold {
        font-weight: bold;
      }
    }
    ```

File: dapps/organizations/src/components/detail/identifications/notary/request/request.vue [hunk 9]
```diff
@@ -49,34 +49,7 @@
...
-                  </div>
-                </div>
-              </div>
-
-              <h4 class="mt-4 mb-4">{{'_org.ident.notary.request.costs.title' | translate}}</h4>
-              <p class="mb-4">{{'_org.ident.notary.request.costs.description' | translate}}</p>
+             <info-content />
             </div>
 
             <!-- request verification form -->
```
Comment (Tschuck) [inline dapps/organizations/src/components/detail/identifications/notary/request/request.vue:55]: postal code is missing in submit dialog

image

File: dapps/organizations/src/i18n/en.ts (hunk 3 above)
Comment (karladler) [reply dapps/organizations/src/i18n/en.ts:225]: `notarial verification` is OK, I wouldn't change that as long as it isn't a request from PO.

**Review by Tschuck — COMMENTED** [review]
- please have a look at the comments
- reload button in overview does not work

**Review by Tschuck — COMMENTED** [review]
everything is fine! :)

**Pull request merged**